docs/Manual/07 Plans - Vectorization and Parallelization.md

```diff
@@ -47,9 +47,9 @@ for j in range(5):
 And, of course, we can also unroll both dimensions, removing for-loops completely.
 
 ## `vectorize`
-Many modern and advanced target platforms support SIMD vector instructions. These instructions perform the same operation on an entire vector of elements, all at once. By default, each dimension of an iteration space becomes a for-loop. However, the `vectorize` instruction labels a dimension for vectorized execution, rather than for-looping.  
+Modern target platforms support SIMD vector instructions. These instructions perform the same operation on an entire vector of elements, all at once. By default, each dimension of an iteration space becomes a for-loop. However, the `vectorize` instruction labels a dimension for vectorized execution, rather than for-looping.  
```

Comment:
Instead of "However, the `vectorize` instruction ... ", how about "Instead, the `vectorize` instruction ... "?

Reply:
I think "Instead" won't make sense here. Please reconsider. 

How about we use any of the following?
" The `vectorize` instruction instead labels a dimension for vectorized execution, rather than for-looping"
or
"The `vectorize` instruction on the other hand labels a dimension for vectorized execution, rather than for-looping"
or 
"The `vectorize` instruction however labels a dimension for vectorized execution, rather than for-looping"

Reply:
I like the first one, thanks!